apps/src/lib/cli.rs

```diff
@@ -2192,6 +2227,66 @@ pub mod args {
         }
     }
 
+    #[derive(Debug, Clone)]
+    pub struct ValidatorSetUpdateRelay {
```

Comment:
How does Ethereum know which address to charge the gas from?

Reply:
in hardhat, it always selected an account configured by default, but you are right, there should be an option to change the default wallet to another addr